src/assets/defaults.ts

```diff
@@ -16,7 +16,8 @@ const defaultMiniWidgetManagerVars = {
   configMenuOpen: false,
 }
 
-export const defaultGlobalAddress = window.location.hostname
+const hostname = window.location.hostname
+export const defaultGlobalAddress = hostname == '' || hostname == undefined ? 'blueos-avahi.local' : hostname
```

Comment:
> What's the difference between the traditional blueos.local to this blueos-avahi.local? Does this work on any BlueOS-extension usage, like wired or wireless?

`bueos-avahi.local` is broadcast by all interfaces agains `blueos.local` that's only available over the cable.

> I ask because we had problems with this previously and stayed with window.location.hostname as it was working on all situations.

This will not change the default behaviour, only in the case where hostname does not exist.

> In the electron case, is this variable null?

It's an empty string, this happens inside electron where hostname does not exist since the source code comes from a file endpoint.